### PR TITLE
[SHELL32] fr-fr translation

### PR DESCRIPTION
**Pull request by KyleKatarn31**
## Purpose

_ Translation of all fr-fr missing elements in Shell32 (in particular files & folders options & file associations panels)

## Proposed changes

shell32-2
shell32-3
shell32-1

### CODE REVIEW

File: dll/win32/shell32/lang/fr-FR.rc [hunk 1]
```diff
@@ -639,74 +639,74 @@ système ou le rendre moins fonctionnel.\n\n\
 END
 
 IDD_NEWEXTENSION DIALOGEX 0, 0, 260, 75
-CAPTION "Create New Extension"
+CAPTION "Créer une nouvelle extension"
 STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_CAPTION | WS_SYSMENU
 FONT 8, "MS Shell Dlg"
 BEGIN
-    LTEXT "&File Extension:", -1, 10, 10, 85, 14
+    LTEXT "Extension du fichier", -1, 10, 10, 85, 14
```
Comment (katahiromz) [inline dll/win32/shell32/lang/fr-FR.rc:646]: colon

Comment (KyleKatarn31) [reply dll/win32/shell32/lang/fr-FR.rc:646]: I fix

**Comment by KyleKatarn31** [comment]
@SergeGautherie @sdever @HBelusca : For French Review + Merge if OK.
Thanks

File: dll/win32/shell32/lang/fr-FR.rc [hunk 2]
```diff
@@ -615,10 +615,10 @@ STYLE DS_SHELLFONT | DS_MODALFRAME | DS_CONTEXTHELP | WS_POPUPWINDOW | WS_VISIBL
 CAPTION "Changer l'icône"
 FONT 8, "MS Shell Dlg", 400, 0, 0x1
 BEGIN
-    LTEXT "Look for icons in this file:", -1, 7, 7, 179, 10
+    LTEXT "Rechercher un icône pour ce fichier :", -1, 7, 7, 179, 10
```
Comment (sdever) [inline dll/win32/shell32/lang/fr-FR.rc:618]: une icône ?

File: dll/win32/shell32/lang/fr-FR.rc [hunk 3]
```diff
@@ -615,10 +615,10 @@ STYLE DS_SHELLFONT | DS_MODALFRAME | DS_CONTEXTHELP | WS_POPUPWINDOW | WS_VISIBL
...
     PUSHBUTTON "Parcourir...", IDC_BUTTON_PATH, 135, 17, 50, 14
     EDITTEXT IDC_EDIT_PATH, 7, 17, 123, 14, ES_AUTOHSCROLL
-    LTEXT "Select an icon from the list below:", -1, 7, 36, 179, 10
+    LTEXT "Sélectionnez un icône dans la liste ci-dessous :", -1, 7, 36, 179, 10
```
Comment (sdever) [inline dll/win32/shell32/lang/fr-FR.rc:621]: cf premier commentaire

File: dll/win32/shell32/lang/fr-FR.rc [hunk 4]
```diff
@@ -809,9 +809,9 @@ BEGIN
     IDS_SHUTDOWN_PROMPT "Voulez-vous fermer la session ReactOS ?"
     
     /* Format Dialog Strings */
-    IDS_FORMAT_TITLE "Format Local Disk"
-    IDS_FORMAT_WARNING "WARNING: Formatting will erase ALL data on this disk.\nTo format the disk, click OK. To quit, click CANCEL."
-    IDS_FORMAT_COMPLETE "Format Complete."
+    IDS_FORMAT_TITLE "Formater un disque logique"
+    IDS_FORMAT_WARNING "Attention: Le formatage effacera toutes les données présentes sur ce disque.\nPour formater ce disque, clickez sur OK. Pour quitter, clickez sur Annuler."
```
Comment (sdever) [inline dll/win32/shell32/lang/fr-FR.rc:813]: "clickez sur OK" -> "cliquez sur OK"

File: dll/win32/shell32/lang/fr-FR.rc [hunk 5]
```diff
@@ -948,40 +948,40 @@ BEGIN
...
-    IDS_ACTION_EXISTS "The action '%s' is already registered for this file type. Please enter a different name and try again."
-    IDS_EXE_FILTER "Programs\0*.exe\0All Files\0*.*\0"
-    IDS_EDITING_ACTION "Editing action for type: "
-    IDS_NO_ICONS "The file '%s' contains no icons.\n\nChoose an icon from the list or specify a different file."
-    IDS_FILE_NOT_FOUND "The file '%s' was not found."
+    IDS_ADVANCED_FOLDER "Fichiers et dossiers"
+    IDS_ADVANCED_NET_CRAWLER "Rechercher automatiquement les dossiers et imprimantes partagés"
+    IDS_ADVANCED_FOLDER_SIZE_TIP "Afficher les informations concernant la taille des fichiers dans les info-bulles"
+    IDS_ADVANCED_FRIENDLY_TREE "Afficher une vue simple des dossiers dans la liste des dossiers de l'Explorateur ReactOS"
+    IDS_ADVANCED_WEB_VIEW_BARRICADE "Affichier le contenu des dossiers système"
```
Comment (sdever) [inline dll/win32/shell32/lang/fr-FR.rc:955]: "Afficher", not "Affichier"

File: dll/win32/shell32/lang/fr-FR.rc [hunk 6]
```diff
@@ -948,40 +948,40 @@ BEGIN
...
+    IDS_ADVANCED_SHOW_FULL_PATH_ADDRESS "Afficher le chemin complet dans la barre d'adresse"
```
Comment (sdever) [inline dll/win32/shell32/lang/fr-FR.rc:956]: barre d'adresses

File: dll/win32/shell32/lang/fr-FR.rc [hunk 7]
```diff
@@ -948,40 +948,40 @@ BEGIN
...
+    IDS_ADVANCED_HIDDEN "Fichiers et dossiers cachés"
+    IDS_ADVANCED_DONT_SHOW_HIDDEN "Ne pas afficher les fichiers et dossiers cachés"
+    IDS_ADVANCED_SHOW_HIDDEN "Afficher les fichiers et dossiers cachés"
+    IDS_ADVANCED_HIDE_FILE_EXT "Masquer les extensions des fichiers dont le type est connu"
+    IDS_ADVANCED_SUPER_HIDDEN "Masquer les fichiers protégés du système d'exploitation (recommandé)"
+    IDS_ADVANCED_DESKTOP_PROCESS "Ouvrir les fenêtres des dossiers dans un processus différent"
+    IDS_ADVANCED_CLASSIC_VIEW_STATE "Mémoriser les paramètres d'affichage de chaque dossier"
+    IDS_ADVANCED_PERSIST_BROWSERS "Restaurer les fenêtres de dossiers ouvertes lors de la prochaine ouverture de session"
+    IDS_ADVANCED_CONTROL_PANEL_IN_MY_COMPUTER "Afficher le Panneau de configuration dans le Poste de travail"
+    IDS_ADVANCED_SHOW_COMP_COLOR "Afficher les dossiers et les fichiers NTFS cryptés ou compressés en couleur"
```
Comment (sdever) [inline dll/win32/shell32/lang/fr-FR.rc:968]: "chiffrés" is the right term in cryptography :p

File: dll/win32/shell32/lang/fr-FR.rc [hunk 8]
```diff
@@ -948,40 +948,40 @@ BEGIN
...
+    IDS_ADVANCED_SHOW_INFO_TIP "Afficher les info-bulles pour les fichiers et les dossiers"
+
+    IDS_NEWEXT_ADVANCED_LEFT "<< Avancé"
+    IDS_NEWEXT_ADVANCED_RIGHT "Avancé >>"
+    IDS_NEWEXT_NEW "<Nouveau>"
+    IDS_NEWEXT_SPECIFY_EXT "Vous devez spécifier une extension."
+    IDS_NEWEXT_ALREADY_ASSOC "L'extension %s est déjà associée avec le type de fichiers %s. Voulez vous désassocier %s avec %s et créer une nouvelle association de fichier pour celle-ci ?"
+    IDS_NEWEXT_EXT_IN_USE "Extensions utilisées"
+
+    IDS_REMOVE_EXT "Si vous retirez une extension enregistrée, il ne sera plus possible d'ouvrir les fichiers avec cette extension en double clickant sur leur icône.\n\nÊtes-vous sûr de vouloir retirer cette extension ?"
```
Comment (sdever) [inline dll/win32/shell32/lang/fr-FR.rc:978]: double-cliquant

File: dll/win32/shell32/lang/fr-FR.rc (hunk 2 above)
Comment (KyleKatarn31) [reply dll/win32/shell32/lang/fr-FR.rc:618]: OK

File: dll/win32/shell32/lang/fr-FR.rc (hunk 4 above)
Comment (KyleKatarn31) [reply dll/win32/shell32/lang/fr-FR.rc:813]: OK

File: dll/win32/shell32/lang/fr-FR.rc (hunk 6 above)
Comment (KyleKatarn31) [reply dll/win32/shell32/lang/fr-FR.rc:956]: Checked. No change (ok with @HBelusca )

File: dll/win32/shell32/lang/fr-FR.rc (hunk 8 above)
Comment (KyleKatarn31) [reply dll/win32/shell32/lang/fr-FR.rc:978]: OK

File: dll/win32/shell32/lang/fr-FR.rc (hunk 7 above)
Comment (KyleKatarn31) [reply dll/win32/shell32/lang/fr-FR.rc]: fixed

File: dll/win32/shell32/lang/fr-FR.rc (hunk 5 above)
Comment (KyleKatarn31) [reply dll/win32/shell32/lang/fr-FR.rc]: fixed

File: dll/win32/shell32/lang/fr-FR.rc (hunk 3 above)
Comment (KyleKatarn31) [reply dll/win32/shell32/lang/fr-FR.rc]: fixed

File: dll/win32/shell32/lang/fr-FR.rc [hunk 9]
```diff
@@ -639,74 +639,74 @@ système ou le rendre moins fonctionnel.\n\n\
...
     COMBOBOX IDC_FOLDERCUST_COMBOBOX, 15, 35, 210, 300, CBS_HASSTRINGS | CBS_AUTOHSCROLL | CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
-    AUTOCHECKBOX "Also apply this template to all &subfolders", IDC_FOLDERCUST_CHECKBOX, 15, 50, 210, 15
-    GROUPBOX "Folder pictures", IDC_STATIC, 5, 75, 230, 90, WS_TABSTOP
-    LTEXT "For Thumbnails view, you can put a picture on this folder to remind you of the contents.", IDC_STATIC, 15, 87, 115, 33
-    PUSHBUTTON "Choose &Picture...", IDC_FOLDERCUST_CHOOSE_PIC, 15, 125, 115, 15
-    PUSHBUTTON "&Restore Default", IDC_FOLDERCUST_RESTORE_DEFAULTS, 15, 144, 115, 15
-    LTEXT "Preview:", IDC_STATIC, 139, 85, 81, 11
+    AUTOCHECKBOX "Appliquer ce modèle aux sous-dossiers", IDC_FOLDERCUST_CHECKBOX, 15, 50, 210, 15
+    GROUPBOX "Images de dossier", IDC_STATIC, 5, 75, 230, 90, WS_TABSTOP
+    LTEXT "Pour la vue des minuatures, vous pouvez mettre une image sur ce dossier pour vous souvenir de son contenu.", IDC_STATIC, 15, 87, 115, 33
```
Comment (sdever) [inline dll/win32/shell32/lang/fr-FR.rc:701]: Miniatures

File: dll/win32/shell32/lang/fr-FR.rc [hunk 10]
```diff
@@ -742,10 +742,10 @@ BEGIN
     IDS_ADMINISTRATIVETOOLS "Outils d'administration"
 
     /* special folders descriptions */
-    IDS_ADMINISTRATIVETOOLS_DESCRIPTION "Configure administrative settings for your computer."
-    IDS_FOLDER_OPTIONS_DESCRIPTION "Customize the display of files and folders and change file associations."
-    IDS_FONTS_DESCRIPTION "Add, change, and manage fonts on your computer."
-    IDS_PRINTERS_DESCRIPTION "Shows installed printers and fax printers and helps you add new ones."
+    IDS_ADMINISTRATIVETOOLS_DESCRIPTION "Configurer les paramètres administratifs pour cet ordinateur."
+    IDS_FOLDER_OPTIONS_DESCRIPTION "Personnaliser l'affichage des fichiers et dossier et modifier les associates de fichiers."
```
Comment (sdever) [inline dll/win32/shell32/lang/fr-FR.rc:746]: associations de fichier

File: dll/win32/shell32/lang/fr-FR.rc (hunk 9 above)
Comment (KyleKatarn31) [reply dll/win32/shell32/lang/fr-FR.rc:701]: fixed

File: dll/win32/shell32/lang/fr-FR.rc (hunk 10 above)
Comment (KyleKatarn31) [reply dll/win32/shell32/lang/fr-FR.rc:746]: fixed

File: dll/win32/shell32/lang/fr-FR.rc [hunk 11]
```diff
@@ -639,74 +639,74 @@ système ou le rendre moins fonctionnel.\n\n\
...
-    LTEXT "For Thumbnails view, you can put a picture on this folder to remind you of the contents.", IDC_STATIC, 15, 87, 115, 33
-    PUSHBUTTON "Choose &Picture...", IDC_FOLDERCUST_CHOOSE_PIC, 15, 125, 115, 15
-    PUSHBUTTON "&Restore Default", IDC_FOLDERCUST_RESTORE_DEFAULTS, 15, 144, 115, 15
-    LTEXT "Preview:", IDC_STATIC, 139, 85, 81, 11
+    AUTOCHECKBOX "Appliquer ce modèle aux sous-dossiers", IDC_FOLDERCUST_CHECKBOX, 15, 50, 210, 15
+    GROUPBOX "Images de dossier", IDC_STATIC, 5, 75, 230, 90, WS_TABSTOP
+    LTEXT "Pour la vue des miniatures, vous pouvez mettre une image sur ce dossier pour vous souvenir de son contenu.", IDC_STATIC, 15, 87, 115, 33
+    PUSHBUTTON "Choisir une image...", IDC_FOLDERCUST_CHOOSE_PIC, 15, 125, 115, 15
+    PUSHBUTTON "&Par défaut", IDC_FOLDERCUST_RESTORE_DEFAULTS, 15, 144, 115, 15
+    LTEXT "Apperçu :", IDC_STATIC, 139, 85, 81, 11
```
Comment (sdever) [inline dll/win32/shell32/lang/fr-FR.rc:704]: Aperçu

File: dll/win32/shell32/lang/fr-FR.rc [hunk 12]
```diff
@@ -742,10 +742,10 @@ BEGIN
     IDS_ADMINISTRATIVETOOLS "Outils d'administration"
 
     /* special folders descriptions */
-    IDS_ADMINISTRATIVETOOLS_DESCRIPTION "Configure administrative settings for your computer."
-    IDS_FOLDER_OPTIONS_DESCRIPTION "Customize the display of files and folders and change file associations."
-    IDS_FONTS_DESCRIPTION "Add, change, and manage fonts on your computer."
-    IDS_PRINTERS_DESCRIPTION "Shows installed printers and fax printers and helps you add new ones."
+    IDS_ADMINISTRATIVETOOLS_DESCRIPTION "Configurer les paramètres administratifs pour cet ordinateur."
+    IDS_FOLDER_OPTIONS_DESCRIPTION "Personnaliser l'affichage des fichiers et dossier et modifier les associations de fichiers."
```
Comment (sdever) [inline dll/win32/shell32/lang/fr-FR.rc:746]: fichiers et dossiers

File: dll/win32/shell32/lang/fr-FR.rc (hunk 11 above)
Comment (KyleKatarn31) [reply dll/win32/shell32/lang/fr-FR.rc:704]: ok

File: dll/win32/shell32/lang/fr-FR.rc (hunk 12 above)
Comment (KyleKatarn31) [reply dll/win32/shell32/lang/fr-FR.rc:746]: ok

**Pull request merged**